### PR TITLE
cli: update examples that refer to --version

### PR DESCRIPTION
**Pull request by zmalik**
**What this PR does / why we need it**:
- add examples to help users with `--operator-version` and `--app-version`
- adds also `--operator-version` and `--app-version` to install flags section
- changes flink demo versions so they are working with `--operator-version` and new versions

### CODE REVIEW

File: content/docs/cli.md [hunk 1]
```diff
@@ -126,8 +130,8 @@ The file path to Kubernetes configuration file. (default: "$HOME/.kube/config")
 The namespace used for the operator installation. (default: "default")
 :::
 
-::: flag --package-version (string)
-A specific package version on the official GitHub repo. (default to the most recent)
+::: flag --operator-version (string)
+A specific operator version int the official GitHub repo. (default to the most recent)
```
Comment (mpereira) [inline content/docs/cli.md]:
```suggestion
A specific operator version in the official GitHub repo. (default to the most recent)
```

File: content/docs/cli.md [hunk 2]
```diff
@@ -147,7 +151,7 @@ KUDO itself is a Kubernetes operator. As such it requires the installation of CR
 
 * `kubectl kudo init --wait --wait-timeout 600` which will install CRDS, install KUDO and will wait up to 600 seconds for KUDO to be responsive.
 * `kubectl kudo init --dry-run --output=yaml > kudo-install.yaml` which will not install anything but will output YAML to a file which can be applied manually to the server.
-* `kubectl kudo init --version=0.5.0` which will install the `0.5.0` into the cluster using the image `kudobuilder/controller:v0.5.0`
+* `kubectl kudo init --version=0.10.0` which will install the `0.10.0` into the cluster using the image `kudobuilder/controller:v0.5.0`
```
Comment (mpereira) [inline content/docs/cli.md]:
```suggestion
* `kubectl kudo init --version=0.10.0` which will install the `0.10.0` version in the cluster using the `kudobuilder/controller:v0.10.0` image
```

**Review by mpereira — APPROVED** [review]
Thanks @zmalik.

File: content/docs/cli.md [hunk 3]
```diff
@@ -183,6 +187,18 @@ kubectl kudo install kafka
...
+kubectl kudo install kafka --app-version=2.4.0
+```
+
+This will install new instance of Kafka Operator which maps to Kafka app version 2.4.0
+
+```bash
+kubectl kudo install kafka --operator-version=1.2.0
+```
+
+This will install new instance ofq Kafka Operator which maps to Kafka operator version 1.2.0
```
Comment (meichstedt) [inline content/docs/cli.md]:
```suggestion
This will install new instance of Kafka Operator which maps to Kafka operator version 1.2.0
```

**Review by meichstedt — APPROVED** [review]
suggested a typo fix, otherwise lgtm!

**Pull request merged**